src/api/app/views/webui/monitor/old.html.erb

```diff
@@ -1,5 +1,7 @@
 <% @pagetitle = 'Build Status Monitor' %>
-<% @crumb_list = %w(Monitor) %>
+<% @crumb_list = [] %>
+<% @crumb_list << link_to('Monitor', :controller => 'monitor', :action => :index) %>
+<% @crumb_list << %w(Detailed) %>
```

Comment:
Ok I'll fix that. My change does work in the webUI, which is why I didn't notice anything wrong  with it.